src/app/modules/server-settings/pages/manage-settings/manage-settings.component.ts

```diff
@@ -55,6 +55,21 @@ export class ManageSettingsComponent implements OnInit, OnDestroy {
                     return null;
                 }
             ]),
+            maxUploadSize: new FormControl('unlimited', [
+              Validators.required,
+              Validators.pattern('^unlimited$|([1-9][0-9]*[kmgKMG]?[bB])'),
+              (control: FormControl) => {
+                const message = {
+                  invalidSize: 'Max file upload size should be: "unlimited" or for example 100MB'
+                };
+
+                if (control.invalid) {
+                  return message;
+                }
+
+                return null;
+              }
+            ]),
```

Comment:
Cosmetic: Follow the code style which is `4 white spaces` (you seem to be using 2)